src/app/views/components/timeline-view/timeline-view.component.ts

```diff
@@ -498,23 +552,40 @@ export class TimelineViewComponent implements OnInit, AfterViewInit, OnDestroy {
   onBreadcrumbClick(breadcrumb: TimelineBreadcrumb) {
     this.showFolderDetails = false;
     if (breadcrumb.type === 'folder') {
-      if (breadcrumb.folder_linkId === this.dataService.currentFolder.folder_linkId) {
+      if (
+        breadcrumb.folder_linkId ===
+        this.dataService.currentFolder.folder_linkId
+      ) {
         this.groupTimelineItems(true, false);
+        this.addPixelMargin();
         this.timeline.fit();
         this.breadcrumbs.setTimeBreadcrumbs();
       } else {
-        this.onFolderClick(new FolderVO({
-          archiveNbr: breadcrumb.archiveNbr,
-          folder_linkId: breadcrumb.folder_linkId
-        }));
+        this.onFolderClick(
+          new FolderVO({
+            archiveNbr: breadcrumb.archiveNbr,
+            folder_linkId: breadcrumb.folder_linkId,
+          })
+        );
       }
     } else {
       const groups = this.timelineGroups.get(breadcrumb.timespan);
       const group = find(groups, (g: TimelineGroup | TimelineItem) => {
-        return g.dataType === 'group' && g.content === breadcrumb.text ;
+        return g.dataType === 'group' && g.content === breadcrumb.text;
       });
       this.onGroupClick(group as TimelineGroup);
     }
   }
 
+  protected addPixelMargin(): void {
```

Comment:
Just out of curiosity, what is it that forces us to include the `@ts-ignore`s here?